### DISCUSSION TITLE
Pull requestsに対するレビューの仕方

### DISCUSSION THREAD

**Discussion by yuukimaru130516** [satsukizzz/n-playspace-http#11] [Show and tell]
## Pull requestsに対するレビューの仕方

まず、レビューしたいプルリクにアクセスして**Files changed**をクリックします。
そうすると、右上に緑色で**Review changes**というボタンがあるので、それをクリックします。
（クリック後が以下の添付画像です）

![スクリーンショット (163)](https://user-images.githubusercontent.com/64817732/136636925-ed2c8f5f-01d0-4234-a535-a261503d6215.png)

### レビューには3種類あります
1. Comment
1. Approve
1. Request changes

#### Comment
Commentは、単純にフィードバックだけを送りたい場合に使います。

#### Approve
さきほどのCommentniに加えて、変更を承認する場合に使います。今回のレポジトリでは2人以上にApproveされることでマージできるようになります。

#### Request changes
修正依頼がある場合に使います。間違えてるところやバグが生まれそうなところがあったら使います。

**基本的に、修正の必要が無かったらApproveを選択するようにお願いします！**

**Comment by satsukizzz** [comment]
大変助かりますーー！ありがたいです 🙇 🙇

**Comment by kenton116** [comment]
とても助かりました！
ありがとうございます❤️

**Comment by HappyKumamoto** [comment]
どうもありがとうございます！

**Comment by kenton116** [comment]
自分の合作用リポジトリに貼ってもいいですか？
